frontend/device/mass_storage.lua

```diff
@@ -0,0 +1,56 @@
+local Device = require("device")
```

Comment:
This just randomly seems to be here?

Reply:
yup,

leftovers from UsbGadget copy/paste.
I moved the file to ui/elements/mass_storage.lua, since it has nothing to do with device abstraction